keras_retinanet/bin/train.py

```diff
@@ -203,6 +224,12 @@ def parse_args(args):
     pascal_parser = subparsers.add_parser('pascal')
     pascal_parser.add_argument('pascal_path', help='Path to dataset directory (ie. /tmp/VOCdevkit).')
 
+    oid_parser = subparsers.add_parser('oid')
+    oid_parser.add_argument('main_dir', help='Path to dataset directory')
```

Comment:
Bit nitpicky, but it's missing a `.` at the end of the help.

Reply:
Np :)